.github/workflows/lint.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: Linters
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: ['**']
+  pull_request:
```

Comment:
I believe the default behavior of `pull_request` is to run for all three types (`created`, `edited`, `deleted`). Not sure there is much value in running on `deleted`

Reply:
Apparently the default for `pull_request` events is limited to `opened`, `synchronize`, and `reopened`. 
I'm happy to make it more explicit, and any doubt or ambiguity.

Documentation link:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
Oh, I missed that that was the default. Nice!